### ISSUE TITLE
View Reloading not working on 0.8.0

### ISSUE THREAD

**Issue by yjukaku** [hanami/view#111]
Hanami View changes are not being auto-reloaded by the server.

```
user ~/reloadtest $ hanami -v
v0.8.0
user ~/reloadtest $ ruby -v
ruby 2.3.1p112 (2016-04-26 revision 54768) [x86_64-darwin15]
user ~/reloadtest $ bundle show
Gems included by the bundle:
  * addressable (2.4.0)
  * bundler (1.12.5)
  * capybara (2.10.1)
  * concurrent-ruby (1.0.2)
  * dotenv (2.1.1)
  * dry-configurable (0.1.7)
  * dry-container (0.5.0)
  * dry-core (0.1.0)
  * dry-equalizer (0.2.0)
  * dry-logic (0.4.0)
  * dry-types (0.9.0)
  * dry-validation (0.10.3)
  * hanami (0.8.0)
  * hanami-assets (0.3.0)
  * hanami-controller (0.7.1)
  * hanami-helpers (0.4.0)
  * hanami-mailer (0.3.0)
  * hanami-model (0.6.1)
  * hanami-router (0.7.0)
  * hanami-utils (0.8.0)
  * hanami-validations (0.6.0)
  * hanami-view (0.7.0)
  * http_router (0.11.2)
  * inflecto (0.0.2)
  * mail (2.6.4)
  * mime-types (3.1)
  * mime-types-data (3.2016.0521)
  * mini_portile2 (2.1.0)
  * minitest (5.9.1)
  * nokogiri (1.6.8.1)
  * rack (1.6.4)
  * rack-test (0.6.3)
  * rake (11.3.0)
  * sequel (4.39.0)
  * shotgun (0.9.2)
  * thor (0.19.1)
  * tilt (2.0.5)
  * url_mount (0.2.1)
  * xpath (2.0.0)
```

Steps to reproduce:
1. Create new Hanami project. `hanami new reloadtest`
2. Create action. `hanami generate action web books`
3. Start server. `hanami serve`
4. View empty page at `http://localhost:2300/books`
5. Update created view. 

``` ruby
module Web::Views::Books
  class Index
    include Web::View
    def from_view
      123
    end
  end
end
```
1. Change template `books/index.html.erb` to use view method:

``` ruby
# books/index.html.erb
<%= from_view %>
```
1. Crash:
   ![screen shot 2016-10-13 at 1 09 01 pm](https://cloud.githubusercontent.com/assets/908365/19359014/3eddb500-9146-11e6-88d2-ea024445afa8.png)

Note: Killing and then starting the server, then pressing refresh allows the page to render successfully.

Please let me know if you need any more information from me. 👍

**Comment by yjukaku** [comment]
It seems like changing the view template does reload automatically, but not the view object itself.

**Comment by TiteiKo** [comment]
@yjukaku Hi, 0.8.0 introduced this regression that was fixed here https://github.com/hanami/hanami/pull/628 and published with 0.9.0
